frontend/packages/console-app/src/__tests__/network-policies/network-policy-form.spec.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { Alert, FormFieldGroupExpandable } from '@patternfly/react-core';
+import { mount } from 'enzyme';
+import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
+import { NetworkPolicyForm } from '../../components/network-policies/network-policy-form';
+
+jest.mock('react-i18next', () => {
```

Comment:
We have a global mock for this now that you should be able use: https://github.com/openshift/console/blob/master/frontend/__mocks__/react-i18next.ts